dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+import {
+    delay,
+    TEST_QUERIES,
+    querySearch,
+    landOnEventVisualizations
+  } from '../../utils/event_constants';
+  
+  const numberOfWindow = 4;
+  const legendSize = 20;
+  const pointSize = 30;
+  const pointSizeUpdated = 35;
+  const lineWidth = 7;
+  const lineWidthUpdated = 9;
+  const fillOpacity = 10;
+  const fillOpacityUpdated = 50;
+  const rotateLevel = 45;
+  const thresholdValue = 50;
+
+  const renderScatterChart = () => {
+    landOnEventVisualizations();
+    querySearch(TEST_QUERIES[6].query, TEST_QUERIES[6].dateRangeDOM);
+    cy.get('[data-test-subj="configPane__vizTypeSelector"] [data-test-subj="comboBoxInput"]').type('scatter').type('{enter}');
+  };
+  
+  describe('Render scatter chart and verify default behaviour ', () => {
+    beforeEach(() => {
+      renderScatterChart();
+    });
+  
+    it('Render scatter chart and verify by default the data gets render', () => {
+      cy.get('.xy').should('exist');
+    });
+  
+    it('Render scatter chart and verify you see data configuration panel and chart panel', () => {
+      cy.get('.euiPanel.euiPanel--paddingSmall').should('have.length', numberOfWindow);
```

Comment:
Same here: can we add data-test-subj?

Reply:
This element don't have data-test-subj defined, only class can be used here